CHANGELOG.md

```diff
@@ -5,47 +5,60 @@ versioning. Rather than a static releases, this repository contains of a number
 of regression tests that are each semi-independent.  This CHANGELOG file should be used
 to document pull requests to this repository.
 
-### 2025-04-09 ([#159](https://github.com/nasa/harmony-regression-tests/pull/159))
+## 2025-04-15 ([#161](https://github.com/nasa/harmony-regression-tests/pull/161))
+
+### Removed
+
+- NetCDF-to-Zarr regression test suite, due to pending service deprecation.
+- AWS credentials within Docker images, as they were only used by NetCDF-to-Zarr.
+
+### Changed
+
+- Pinned `harmony-py` dependency for Swath Projector and Variable Subsetter test
+  suites. Updated utility functions for each to use updated import paths for
+  `ProcessingFailedException`.
+
+## 2025-04-09 ([#159](https://github.com/nasa/harmony-regression-tests/pull/159))
 
 - Update reference files in SAMBAH test case to reflect updates to Harmony UAT processing.
 
-### 2025-03-21 ([#156](https://github.com/nasa/harmony-regression-tests/pull/156))
+## 2025-03-21 ([#156](https://github.com/nasa/harmony-regression-tests/pull/156))
```

Comment:
The changes from here down in the `CHANGELOG` are just making the header markdown consistent throughout the file. (I'm not OCD, honest 😅)